public/app/dsp-stages.js

```
@@ -0,0 +1,516 @@
+ * Apply a named DSP stage to magnitude/phase arrays.
+ *
+ * @param {string}       stageName  - One of the keys in STAGE_MAP
+ * @param {Float32Array} mag        - Magnitude spectrum (modified in place)
+ * @param {Float32Array} pha        - Phase spectrum
+ * @param {object}       params     - DSP parameter object (merged from slider-map.js)
+ * @param {number}       sampleRate - Audio sample rate (default 44100)
+ */
+export function applyStage(stageName, mag, pha, params = {}, sampleRate = 44100) {
+  const fn = STAGE_MAP[stageName];
+  if (!fn) {
+    console.warn(`[dsp-stages] Unknown stage: '${stageName}'`);
```

ESLint

[error] 'console' is not defined.

(no-undef)